src/OpenTelemetry/OpenTelemetryController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Azure.Functions.PowerShellWorker.PowerShell;
+using Microsoft.Azure.Functions.PowerShellWorker.Utility;
+using LogLevel = Microsoft.Azure.WebJobs.Script.Grpc.Messages.RpcLog.Types.Level;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.OpenTelemetry
+{
+    using PowerShell = System.Management.Automation.PowerShell;
```

Comment:
Are you creating a new PowerShell instance to import the module? Ideally, it would be better if we could use the first PowerShell instance to do this. This way, we wouldn't have to initialize two instances, thereby avoiding a higher cold start for this code path.

Reply:
This could be addressed later as a bug fix. Please open an issue and link it here.

Reply:
No, it uses the PowerShell instance that is about to execute the function. This "using" statement is because the namespace Microsoft.Azure.Functions.PowerShellWorker.PowerShell exists, and so when I try to use the type PowerShell from System.Management.Automation, it considers it ambiguous without